Formula/x/xk6.rb

```diff
@@ -0,0 +1,18 @@
+class Xk6 < Formula
+  desc "Build k6 with extensions"
+  homepage "https://k6.io"
+  url "https://github.com/grafana/xk6/archive/refs/tags/v0.14.3.tar.gz"
+  sha256 "35a7fa8e059caf7848a67d7dd271f754ef06cf52512b08da1134971b39004900"
+  license "Apache-2.0"
+
+  depends_on "go"
```

Comment:
just need to be build dep?
```suggestion
  depends_on "go" => :build
```

Reply:
Actually, xk6 uses go to build k6, it is a kind of build tool for k6. So it's not just a build dependency.

Reply:
I see, make sense 👍